compiler/lsp-tests/src/Main.hs

```diff
@@ -717,6 +717,16 @@ symbolsTests run =
               foo <- openDoc' "Foo.daml" damlId $ T.unlines ["module Foo where"]
               syms <- getDocumentSymbols foo
               liftIO $ preview (_Left . _head . children . _Just) syms @?= Just (List [])
+        , testCase "no internal imports for module with one template" $
+          run $ do
+              foo <-
+                  openDoc' "Foo.daml" damlId $
+                  T.unlines
+                      ["module Foo where", "template T with p1 : Party where", "  signatory p1"]
```

Comment:
Maybe worth adding a choice in here as well?

Reply:
added a choice.